dodgeball/dodgeball.js

```diff
@@ -0,0 +1,168 @@
+const arrOfPeople = [
+  {
+    name: "Drew Nolan",
+    age: 28,
+    skillSet: "yogi",
+    placeBorn: "San Luis, CA"
+  },
+  {
+    name: "Charles Young",
+    age: 25,
+    skillSet: "welder",
+    placeBorn: "Omaha, NB"
+  },
+  {
+    name: "Judy Twilight",
+    age: 35,
+    skillSet: "fisher",
+    placeBorn: "Louisville, KY"
+  },
+  {
+    name: "Cynthia Doolittle",
+    age: 20,
+    skillSet: "farmer",
+    placeBorn: "Pawnee, TX"
+  },
+  {
+    name: "John Willouby",
+    age: 28,
+    skillSet: "artist",
+    placeBorn: "Manhattan, NY"
+  },
+  {
+    name: "Stan Honest",
+    age: 20,
+    skillSet: "strongman",
+    placeBorn: "Clevland, OH"
+  },
+  {
+    name: "Mia Watu",
+    age: 19,
+    skillSet: "acrobatic",
+    placeBorn: "Los Angeles, CA"
+  },
+  {
+    name: "Walter Cole",
+    age: 32,
+    skillSet: "calistenics",
+    placeBorn: "New Orleans, LA"
+  },
+]
+
+const listOfPlayers = []
+const blueTeam = []
+const redTeam = []
+
+class Player {
+  constructor(name, age, skillSet) {
+    this.name = name
+    this.age = age
+    this.skillSet = skillSet
+    this.placeBorn = this.placeBorn
+  }
+  addToArrayOfPeople() {
+    arrOfPeople.push(this)
+  }
+}
+class Teammate extends Player {
+  constructor (name, age, skillSet, placeBorn, team) {
+    super (name, age, skillSet, placeBorn)
+    this.team = team
+  }
+  joinTeam() {
+    if (this.team === 'blue') {
+      blueTeam.push(this.name)
+    } else if (this.team === 'red') {
+      redTeam.push(this.name)
+    }
+  }
+}
+
+const listPeopleChoices = () => {
+  const listElement = document.getElementById('people')
+  arrOfPeople.map(person => {
+    const li = document.createElement("li")
+    const makePlayerBtn = document.createElement("button")
+    makePlayerBtn.innerHTML = "Make Player"
+    makePlayerBtn.classList.add('makePlayer')
+
+    // Make Player button functionality
+    makePlayerBtn.addEventListener('click', function() {
+      makePlayer(person)
```

Comment:
I see you defined the Player and Teammate classes but you did not instantiate any objects using those classes.